### PR TITLE
[US_CO] Create CO tenant with all views enabled

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

This PR sets up the CO tenant. Right now, it's a copy of the MI tenant and is backed by a copy of the same dummy data. Creating this tenant will allow CO users to log in to staging and (eventually) prod using SSO so we can test the experience during development.

I'm going to use this PR as a base for some more config reorganizing. The goal will be to get as many of the state-specific configs into the single US_XX_CONFIG file as possible.

cc @danawillow for an example of setting up a brand new state in Pathways

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/tenants/US_CO.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/*
+ * Recidiviz - a data platform for criCOnal justice reform
```
Comment (macfarlandian) [inline src/tenants/US_CO.ts:2]: "criMInal" lol

**Review by macfarlandian — APPROVED** [review]
fascinating, I've never actually seen this done before. When you talk about reorganizing, do you mean moving things like `enabledFilters` or `enabledTableColumns` into the state config file?

File: src/tenants/US_CO.ts (hunk 1 above)
Comment (Catacola) [reply src/tenants/US_CO.ts:2]: hahaha, find a replace strikes again

**Comment by Catacola** [comment]
> fascinating, I've never actually seen this done before. When you talk about reorganizing, do you mean moving things like `enabledFilters` or `enabledTableColumns` into the state config file?

Exactly. I'd like to keep all of that in one place per state. I'll need to play around a little bit with naming and structure so everything makes sense and there aren't circular imports.

**Pull request merged**